### DISCUSSION TITLE
Name of Bible Books

### DISCUSSION THREAD

**Discussion by pepaulo** [JenuelDev/Believers-Sword#64] [Ideas]
The app don't translate the Bible Books names in other language. And don't show the deuterocanonical books.

**Comment by JenuelDev** [comment]
There is a change in version `0.4.3`. We now have an option to show Deuterocanonical books by enabling it in the settings.
![image](https://github.com/Bible-Projects/believers-sword-next/assets/31676163/c33fb373-09c4-4e31-bedd-2ed57aa238a5)
